Subject: [PATCH] Set special D-Bus service to messagebus.service
From: Andrey Borzenkov <arvidjaar@mail.ru>

As long as main services are initscript based, set special D-Bus
service name to messagebus.service so that systemd will recognize
it is started and reister as appropriate.

Signed-off-by: Andrey Borzenkov <arvidjaar@mail.ru>

---

 src/special.h |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/src/special.h b/src/special.h
index df1e1fa..96e1cab 100644
--- a/src/special.h
+++ b/src/special.h
@@ -61,7 +61,7 @@
 #define SPECIAL_POWEROFF_TARGET "poweroff.target"
 #define SPECIAL_REBOOT_TARGET "reboot.target"
 #define SPECIAL_KEXEC_TARGET "kexec.target"
-#define SPECIAL_DBUS_SERVICE "dbus.service"
+#define SPECIAL_DBUS_SERVICE "messagebus.service"
 #define SPECIAL_DBUS_SOCKET "dbus.socket"
 #define SPECIAL_GETTY_TARGET "getty.target"
 #define SPECIAL_SERIAL_GETTY_SERVICE "serial-getty@.service"
